caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/GeminiResolver.java

```diff
@@ -85,7 +85,7 @@ private String getPath(URI uri) {
         String fileName = path[1];
         String fileType = FILE_URI;
         String[] fileNameParts = fileName.split("\\.");
-        if (fileName.endsWith("jpg")) {
+        if (fileName.endsWith(".jpg")) {
             fileName = fileNameParts[0] + ".fits"; 
```

Comment:
Still not bulletproof: How will this code handle a filename line f1234.5678.jpg ?

Answer: it will produce f1234.fits